core/types/legacy_tx.go

```diff
@@ -67,15 +67,41 @@ func (ct *CommonTx) GetData() []byte {
 	return ct.Data
 }
 
+func (ct *CommonTx) initializeMisc() {
+	if ct.TransactionMisc == nil {
+		ct.TransactionMisc = &TransactionMisc{}
+	}
+}
+
+func (ct *CommonTx) LoadHash() any {
+	ct.initializeMisc()
+	return ct.hash.Load()
+}
+
+func (ct *CommonTx) StoreHash(val any) {
+	ct.initializeMisc()
+	ct.hash.Store(val)
+}
+
+func (ct *CommonTx) LoadFrom() any {
+	ct.initializeMisc()
+	return ct.from.Load()
+}
+
+func (ct *CommonTx) StoreFrom(val any) {
+	ct.initializeMisc()
+	ct.from.Store(val)
+}
```

Comment:
Shouldn't the receivers be for `TransactionMisc` instead

Reply:
If then, we can't wrap `initializeMisc()`